src/components/AppElements/components/CodeEditor/codeGenerators/emailEditorCodeGenerator.js

```diff
@@ -116,10 +116,12 @@ const calculateAddonsComponents = (editorConfig, indent) => {
 
 	let config = `{
 ${"\t".repeat(indent)}enabled: ${components.enabled},`;
+
 	if (components.disabledReason) config += `\n${ "\t".repeat(indent)}disabledReason: "${editorConfig.settings.addons.components.disabledReason}",`;
-	if (components.behaviour) config += `\n${ "\t".repeat(indent)}behaviour: "${components.behaviour}",`;
-	config += `\n${"\t".repeat(indent - 1)}},`;
+	if (components.behavior) config += `\n${ "\t".repeat(indent)}behavior: "${components.behavior}",`;
+	if (components.behaviorDescription) config += `\n${ "\t".repeat(indent)}behaviorDescription: "${components.behaviorDescription}",`;
```

Comment:
```suggestion
	if (components.behaviorDescription) config += `\n${"\t".repeat(indent)}behaviorDescription: "${components.behaviorDescription}",`;
```
I think the single whitespace before the "\t" messes up the indentation, please remove it from everywhere.
Most likely I messed up when I added the disabledReason to this and it was replicated from that

Reply:
Hey, which indentation messed up? I can't see it.

Reply:
there is an empty space char before the "\t", you can see it in the suggested change that I made